apis/projectcontour/v1alpha1/contourconfig.go

```diff
@@ -532,6 +544,21 @@ const (
 	AllClusterDNSFamily ClusterDNSFamilyType = "all"
 )
 
+// ServerHeaderTransformation defines the action to be applied to the Server header on the response path
+type ServerHeaderTransformationType int32
+
+const (
+	// Overwrite any Server header with the contents of server_name.
+	// This is the default value
+	OVERWRITE ServerHeaderTransformationType = 0
```

Comment:
Let's format these constants like the following (please make similar changes for the other consts as well):
```suggestion
	// Overwrite any Server header with "envoy".
	// This is the default value.
	OverwriteServerHeader ServerHeaderTransformationType = "overwrite"
```